libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/extensions/ContextEx.kt

```diff
@@ -1,6 +1,6 @@
 @file:JvmName("ContextEx")
 
-package com.mapbox.navigation.base.extensions
+package com.mapbox.navigation.base.internal.extensions
```

Comment:
it cannot be _internal_, `examples` uses it

Reply:
`examples` is our test app, we can use it 😉